pages/linux/lvresize.md

```diff
@@ -1,7 +1,8 @@
 # lvresize
 
 > Change the size of a logical volume.
-> More information: <https://man7.org/linux/man-pages/man8/lvresize.8.html>.
```

Comment:
I'm really reluctant to change this URL from man7.org - a neutral site - to redhat.com - a distribution (and distribution version specific) URL. I wasn't aware that man7.org pages were a thing for LVM tools, but now that I am it seems like a much better alternative than the redhat docs.

Reply:
The reason I prefer the RHEL URL is that it provides a conceptual guide. I can always use man if I want the man page.

Reply:
Yeah, @sbrl it's not a link to a man page; it's a link to a guide.  I'm kindof in with @aherst.

Reply:
It doesn't really seem that useful to link to a man page when a good, formal (and relatively official) tutorial exists.  The manpage is good if there's nothing else, but...those are my thoughts.

Reply:
Here's another option. For each of the lvm tools I will set the More information links point to https://man7.org/linux/man-pages/man8/lvm.8.html. That will provide contextualization for each of the tools. When I write the lvm tldr-page I'll set its More information link to the RHEL guide. Comments?

Reply:
Maybe you could keep each respective manpage, but add
```
See also: `lvm`
```
Above the More information line.  I like that the best&mdash;what do you think?

Reply:
OK. "See also: `lvm`" is much more concise than my proposal. 

I'll change the More Information lines to the man page on man7.org. I do still think pointing to a non-man page will add more value. I'll chime in if and when the subject comes up for discussion again. 

I did learn a valuable lesson about not confounding two issues in one PR :-). My original intent was just to make the results returned from "tldr -s lvm" to be exhaustive. Adding and harmonizing the More info lines was an after thought. I'll keep this in mind moving forward.